src/components/TopTenChannels.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from '@emotion/styled'
+import React from 'react'
+
+import { useMostViewedChannels } from '@/api/hooks'
+import { sizes } from '@/shared/theme'
+
+import { ChannelGallery } from './ChannelGallery'
+
+export const TopTenChannels = () => {
+  const { channels, loading } = useMostViewedChannels({ viewedWithinDays: 10, limit: 10 })
```

Comment:
Just FYI, we should use here `useMostFollowedChannels`, but https://github.com/Joystream/orion/pull/17 is not merged yet, so temporarily I used `useMostViewedChannels` here